### ISSUE TITLE
Link to https://github.com/alundesap/python-jws does not work

### ISSUE THREAD

**Issue by jorgelaranjo** [SAP/hana-xsa-python-buildpack#1]
Link to https://github.com/alundesap/python-jws does not work, thus assume that repo is no longer used?
Thus Readme should be updated, I'd say

**Comment by andrewlunde** [comment]
Yes, this was for interim testing.  SAP not provides official Python libs.  Find a post about them here.

https://blogs.sap.com/2018/09/26/shoutout-to-all-pythoneers-python-is-now-among-the-supported-runtimes-in-sap-cloud-platform/#

Thanks for the note.

**Comment by jorgelaranjo** [comment]
@alundesap Yes, I got the `XS-Python` package especially for the `JWT`
You know where the remainder of info of the `JWT` inc which algorithms available for encoding are available by any chance? Or can dig into the `package` code I guess :)

Thanks

**Comment by jonathanbaker7** [comment]
He is correct, though, and the README should still be updated to remove any bad links.

**Comment by jorgelaranjo** [comment]
> @alundesap Yes, I got the `XS-Python` package especially for the `JWT`
> You know where the remainder of info of the `JWT` inc which algorithms available for encoding are available by any chance? Or can dig into the `package` code I guess :)
> 
> Thanks

Documentation is available on the package itself but the example is incorrect as it would try to use `HS256` which is not the supported algorithm.
The correct example would be with `RS256` which works just fine.

Not sure what is the main diff in between the `XS-Python` re `JWToken` and the `python-jws` from which code seems to have been forked from.
